src/controllers/hid/hidcontroller.h

```diff
@@ -40,13 +40,32 @@ class HidController final : public Controller {
 
   private:
     bool isPolling() const override;
+    void processInputReport(int bytesRead);
 
     // For devices which only support a single report, reportID must be set to
     // 0x0.
     void sendBytes(const QByteArray& data) override;
     void sendBytesReport(QByteArray data, unsigned int reportID);
     void sendFeatureReport(const QList<int>& dataList, unsigned int reportID);
 
+    // getInputReport receives an input report on request.
+    // This can be used on startup, to initialize the knob positions in Mixxx
+    // to the physical position of the hardware knobs on the controller.
+    // The returned data structure for the input reports is the same,
+    // as in the polling functionality (incl. ReportID in first byte).
+    // The returned list can be used, to call the incomingData
+    // function of the common-hid-packet-parser.
```

Comment:
```suggestion
    // getInputReport receives an input report on request.
    // This can be used on startup to initialize the knob positions in Mixxx
    // to the physical position of the hardware knobs on the controller.
    // The returned data structure for the input reports is the same
    // as in the polling functionality (including ReportID in first byte).
    // The returned list can be used to call the incomingData
    // function of the common-hid-packet-parser.
```